<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="change_pattern">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить шаблон</string>
   </property>
  </widget>
  <widget class="QLabel" name="pattern_info">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>420</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Текущий шаблон:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pattern_preview">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>440</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="text1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>571</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Настройки текста 1:</string>
   </property>
   <widget class="QLabel" name="text1_XY_info">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущее положение:
X: 0, Y: 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="text1_align">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>19</y>
      <width>120</width>
      <height>81</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Выравнивание по:</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="text1_align_right">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Право</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="text1_align_center">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Центр</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="text1_align_left">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Лево</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="text1_Size">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить размер</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="text1_XY">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить позицию</string>
    </property>
   </widget>
   <widget class="QLabel" name="text1_Size_info">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущий размер:
0x0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="text1_textSize">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="text1_textSize_info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Размер шрифта: </string>
    </property>
   </widget>
   <widget class="QLabel" name="text1_delimiter_info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Слов на строке: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text1_delimiter">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="text2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>571</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Настройки текста 2:</string>
   </property>
   <widget class="QLabel" name="text2_XY_info">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущее положение:
X: 0, Y: 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="text2_align">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>19</y>
      <width>120</width>
      <height>81</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Выравнивание по:</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="text2_align_right">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Право</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="text2_align_center">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Центр</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="text2_align_left">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Лево</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="text2_Size">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить размер</string>
    </property>
   </widget>
   <widget class="QPushButton" name="text2_XY">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить позицию</string>
    </property>
   </widget>
   <widget class="QLabel" name="text2_Size_info">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущий размер:
0x0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="text2_textSize">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="text2_textSize_info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Размер шрифта: </string>
    </property>
   </widget>
   <widget class="QLabel" name="text2_delimiter_info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Слов на строке: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text2_delimiter">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="image1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Настройки картинки 1:</string>
   </property>
   <widget class="QPushButton" name="image1_XY">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить позицию</string>
    </property>
   </widget>
   <widget class="QPushButton" name="image1_Size">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить размер</string>
    </property>
   </widget>
   <widget class="QLabel" name="image1_Size_info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущий размер:
0x0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="image1_XY_info">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущее положение:
X: 0, Y: 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="image2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Настройки картинки 2:</string>
   </property>
   <widget class="QPushButton" name="image2_XY">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить позицию</string>
    </property>
   </widget>
   <widget class="QPushButton" name="image2_Size">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить размер</string>
    </property>
   </widget>
   <widget class="QLabel" name="image2_Size_info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущий размер:
0x0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="image2_XY_info">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущее положение:
X: 0, Y: 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>510</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="creationBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Создание</string>
   </property>
   <widget class="QCheckBox" name="text1_enable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Текст 1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="text2_enable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Текст 2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="image1_enable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Картинка 1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="image2_enable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Картинка 2</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
